src/components/CommentList/CommentList.jsx

```diff
@@ -1,3 +1,15 @@
-export const CommentList = () => (
-  <>Put the list here</>
+import { CommentInfo } from '../CommentInfo';
+import './CommentList.scss';
+
+export const CommentList = ({ comments }) => (
+  <div className="CommentList">
+    {comments.length === 0
```

Comment:
this condition should be at CommentInfo. CommentList just have to get comments and render them